app/src/main/java/com/codingblocks/cbonlineapp/dashboard/DashboardActivity.kt

```diff
@@ -310,7 +304,8 @@ class DashboardActivity : BaseCBActivity(),
             startActivity<LoginActivity>()
             finish()
         } else {
-            startActivity<ProfileActivity>()
+            val args = Pair("user_social_image", vm.prefs.SP_USER_IMAGE)
```

Comment:
Not to be done like this

Reply:
@aggarwalpulkit596 Then what do you suggest here?